fire/FatesFuelMod.F90

```diff
@@ -0,0 +1,363 @@
+          donor_fuel%loading(i)*donor_weight
+        this%frac_loading(i) = this%frac_loading(i)*self_weight +                        &
+          donor_fuel%frac_loading(i)*donor_weight
+        this%frac_burnt(i) = this%frac_burnt(i)*self_weight +                            &
+          donor_fuel%frac_burnt(i)*donor_weight
+        this%effective_moisture(i) = this%effective_moisture(i)*self_weight +            &
+          donor_fuel%effective_moisture(i)*donor_weight
+      end do 
+      
+      this%total_loading = this%total_loading*self_weight +                              &
+        donor_fuel%total_loading*donor_weight
+      this%average_moisture = this%average_moisture*self_weight +                        &
+        donor_fuel%average_moisture*donor_weight
+      this%bulk_density = this%bulk_density*self_weight +                                &
+        donor_fuel%bulk_density*donor_weight      
+      this%SAV = this%SAV*self_weight + donor_fuel%SAV*donor_weight
+      this%MEF = this%MEF*self_weight + donor_fuel%MEF*donor_weight    
+      
+    end subroutine Fuse
+
+    !-------------------------------------------------------------------------------------
+
+    subroutine CalculateLoading(this, leaf_litter, twig_litter, small_branch_litter,     &
+        large_branch_litter, trunk_litter, live_grass)
+      ! DESCRIPTION:
+      !   Calculates loading for each fuel type
+
+      ! ARGUMENTS:
+      class(fuel_type), intent(inout) :: this                ! fuel class
+      real(r8),         intent(in)    :: leaf_litter         ! input leaf litter [kgC/m2]
+      real(r8),         intent(in)    :: twig_litter         ! input twig litter [kgC/m2]
+      real(r8),         intent(in)    :: small_branch_litter ! input small branch litter [kgC/m2]
+      real(r8),         intent(in)    :: large_branch_litter ! input leaf litter [kgC/m2]
+      real(r8),         intent(in)    :: trunk_litter        ! input leaf litter [kgC/m2]
+      real(r8),         intent(in)    :: live_grass          ! input live grass [kgC/m2]
+
+      this%loading(fuel_classes%dead_leaves()) = leaf_litter
+      this%loading(fuel_classes%twigs()) = twig_litter
+      this%loading(fuel_classes%small_branches()) = small_branch_litter
+      this%loading(fuel_classes%large_branches()) = large_branch_litter
+      this%loading(fuel_classes%live_grass()) = live_grass
+      this%loading(fuel_classes%trunks()) = trunk_litter
+
+    end subroutine CalculateLoading
+
+    !-------------------------------------------------------------------------------------
+
+    subroutine SumLoading(this)
+      ! DESCRIPTION:
+      !   Sums up the loading
+
+      ! ARGUMENTS:
+      class(fuel_type), intent(inout) :: this ! fuel class
+      
+      ! LOCALS:
+      integer :: i ! looping index
+      
+      this%total_loading = 0.0_r8
+      do i = 1, nfsc
+        if (i /= fuel_classes%trunks()) then 
+          this%total_loading = this%total_loading + this%loading(i)
```

Comment:
Should this be called "non_trunk_loading"  or "non_trunk_total_loading" ?

Reply:
good idea - it is a bit confusing. I can do that.